docs/getting-started/github-apps/github-apps-landing-page.rst

```diff
@@ -8,22 +8,20 @@ Users can grant a GitHub app specific permissions on the repos and/or organizati
 The Dockstore GitHub App adds support for registering and automatically syncing workflows and services.
 Check out the following guides to learn more:
 
-- :doc:`github-apps`
-- :ref:`Workflow Registration with GitHub Apps <Registration With GitHub Apps>`
+- Start here first! --> :doc:`github-apps`
+- :doc:`Installing the GitHub App </getting-started/github-apps/installation>`
 - :doc:`migrating-workflows-to-github-apps`
```

Comment:
Should we point folks back to how to register a tool or workflow with github apps before "migrating workflows"?